modules/databases/instance.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_db_subnet_group" "default" {
+  name       = "main"
+  subnet_ids = ["${var.subnet}"]
+
+  tags = {
+    Name = "My DB subnet group"
+  }
+}
+
+resource "aws_db_instance" "default" {
+  allocated_storage    = 20
+  storage_type         = "gp2"
+  engine               = "mysql"
+  engine_version       = "5.7"
+  instance_class       = "db.t2.micro"
+  name                 = ""
+  username             = ""
```

Comment:
If parameters are optional and you don't specify them - don't add them. It's useless and makes your code hardly readable.